deps/openlibm.version

```diff
@@ -1,2 +1,2 @@
-OPENLIBM_BRANCH=v0.4.1
-OPENLIBM_SHA1=3ee2a6e6a9fc68cbf2ab00f830b9059215e40b06
+OPENLIBM_BRANCH=master
```

Comment:
I think the technical name would have been `_TREEISH`, but branch seemed a bit clearer. the only real requirement is that the desired sha1 value will always be locatable in the history of tree-ish.

@StefanKarpinski they're just simple string interpolation macros